tests/page_tests.py

```diff
@@ -39,16 +38,16 @@ def strip_between_tags(expression, text):
 
 
 class PageTests(unittest.TestCase):
-    def __init__(self, methodName, ignored_items, wiki_info=None):
+    def __init__(self, method_name, ignored_items, wiki_info: tuple[str, list[str], str, int]):
```

Comment:
The use of the name that the linter doesn't like is because that is the name of the method to be overloaded. See https://docs.python.org/3/library/unittest.html, worth adding a comment in the code to highlight this